### PR TITLE
LPS-197340 Transition Formula Fields Feature Flag to release type

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related ticket: [LPS-197340](https://liferay.atlassian.net/browse/LPS-197340)

### CODE REVIEW

**Review by murilostodolni — APPROVED** [review]
LGTM!

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -6839,6 +6839,21 @@
     #
     feature.flag.LPS-161033.title=Publication History for Individual Assets
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER6__NUMBER4__NUMBER9__NUMBER4__NUMBER8__PERIOD_DESCRIPTION
+    #
+    feature.flag.LPS-164948.description=Enabling this feature will allow users to dynamically populate fields using expressions with information created from data filled and provided by other fields.
```
[the remark this answers is not in the document]
Comment (marcelabc) [reply portal-impl/src/portal.properties:6845]: @murilostodolni @devpedrotavares

[the remark this answers is not in the document]
Comment (murilostodolni) [reply portal-impl/src/portal.properties:6845]: Yes, I totally agree @marcelabc

[the remark this answers is not in the document]
Comment (murilostodolni) [reply portal-impl/src/portal.properties:6845]: I believe we can change the title also: Defining Computed **Object** Fields

**Comment by devpedrotavares** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Comment by devpedrotavares** [comment]
ci:forward

**Comment by devpedrotavares** [comment]
ci:forward:force

**Pull request closed without merging**